homeassistant/components/sensor/enphase_envoy.py

```diff
@@ -81,27 +81,23 @@ def icon(self):
 
     def update(self):
         """Get the energy production data from the Enphase Envoy."""
-        import envoy_reader
+        from envoy_reader import EnvoyReader
 
         if self._type == "production":
```

Comment:
I know this wasn't the case originally, but since you're in here, these strings should be constants, as well.